tools/webdriver/webdriver/bidi/transport.py

```diff
@@ -70,10 +70,13 @@ async def end(self) -> None:
 
     async def read_messages(self) -> None:
         assert self.connection is not None
-        async for msg in self.connection:
-            if not isinstance(msg, str):
-                raise ValueError("Got a binary message")
-            await self.handle(msg)
+        try:
+          async for msg in self.connection:
+              if not isinstance(msg, str):
+                  raise ValueError("Got a binary message")
+              await self.handle(msg)
+        except websockets.ConnectionClosed:
```

Comment:
can this hide some real issues, not only during the teardown? Maybe at least add a log message?

Reply:
I think it should no hide any real issues since the issue will be indicated by the lack of messages or perhaps by the next command being sent. I will add a log